projects/ngneat/error-tailor/src/lib/control-error.directive.ts

```diff
@@ -24,7 +24,8 @@ import { ErrorsMap } from './types';
 
 @Directive({
   selector:
-    '[formControlName]:not([controlErrorsIgnore]), [formControl]:not([controlErrorsIgnore]), [formGroup]:not([controlErrorsIgnore]), [formGroupName]:not([controlErrorsIgnore]), [formArrayName]:not([controlErrorsIgnore]), [ngModel]:not([controlErrorsIgnore])'
+    '[formControlName]:not([controlErrorsIgnore]), [formControl]:not([controlErrorsIgnore]), [formGroup]:not([controlErrorsIgnore]), [formGroupName]:not([controlErrorsIgnore]), [formArrayName]:not([controlErrorsIgnore]), [ngModel]:not([controlErrorsIgnore])',
+  exportAs: 'controlErrorsInstance'
```

Comment:
LGTM. Just change it to `controlErrors`

Reply:
I wanted to differentiate from the same name for `@Input('controlErrors') customErrors: ErrorsMap = {};`. It could be a bit misleading, isn't it?

Reply:
No. It'll be `<input #controlErrors="yournName"`

Reply:
Call it `errorTailor`

Reply:
OK